docs/src/extend/custom-processors.md

```diff
@@ -46,29 +47,28 @@ module.exports = {
 
 **The `preprocess` method** takes the file contents and filename as arguments, and returns an array of code blocks to lint. The code blocks will be linted separately but still be registered to the filename.
 
-A code block has two properties `text` and `filename`; the `text` property is the content of the block and the `filename` property is the name of the block. Name of the block can be anything, but should include the file extension, that would tell the linter how to process the current block. The linter will check [`--ext` CLI option](../use/command-line-interface#--ext) to see if the current block should be linted, and resolve `overrides` configs to check how to process the current block.
+A code block has two properties `text` and `filename`. The `text` property is the content of the block and the `filename` property is the name of the block. The name of the block can be anything, but should include the file extension, which tells the linter how to process the current block. The linter checks the [`--ext` CLI option](../use/command-line-interface#--ext) to see if the current block should be linted and resolves `overrides` configs to check how to process the current block.
```

Comment:
The last sentence here about checking `--ext` describes the pre-ESLint v6 behavior. That still happens, but it also checks extensions in `overrides` patterns now. That's described in https://eslint.org/docs/latest/use/configure/plugins#specify-a-processor.

If updating this is out of scope for the change you're intending here, feel free to disregard.

Reply:
Nice catch! I couldn't remember if we had removed that functionality or not.

Reply:
i don't fully understand. would one of you mind providing a bit more info or a draft of what the copy should say here?

Reply:
thank you for clarifying!

Reply:
To expand on what nzakas said and give an example, with the config below, to use the Markdown plugin prior to ESLint v6, one had to pass `--ext .js,.md`, or ESLint would only look at `.js` files. Now, ESLint will automatically see there's an `overrides` pattern for `.md` files and include them in the file traversal. One can still pass `--ext` to override that automatic behavior, but it in normal cases ESLint will do the right thing automatically.

```
// .eslintrc.js
module.exports = {
    plugins: ["markdown"],
    overrides: [
        {
            files: ["**/*.md"],
            processor: "markdown/markdown"
        }
    ]
};
```

Reply:
I don't think we need anything further.